core/src/main/java/org/apache/carbondata/core/util/DataTypeUtil.java

```diff
@@ -720,21 +721,22 @@ private static String parseStringToBigDecimal(String value, ColumnSchema columnS
     return null;
   }
 
+  public static DataTypeConverter getDataTypeConverter() {
+    if (converter == null) {
+      converter = new DataTypeConverterImpl();
+    }
+    return converter;
+  }
+
```

Comment:
This change does not belong to this PR...the code is already present in the same class...kindly remove all the changes that does not belong to this PR